src/Parser/DeathKnight/Frost/Modules/Features/CastEfficiency.js

```diff
@@ -20,19 +20,26 @@ class CastEfficiency extends CoreCastEfficiency {
       recommendedCastEfficiency: 0.95,
       extraSuggestion: <span>Making sure to use <SpellLink id={SPELLS.SINDRAGOSAS_FURY_ARTIFACT.id}/> immediately after it's cooldown is up is important and if you have 5 stacks of <SpellLink id={SPELLS.RAZORICE.id}/> use as it is coming off cooldown.</span>,
     },
-
     {
       spell: SPELLS.OBLITERATION_TALENT,
       category: CastEfficiency.SPELL_CATEGORIES.COOLDOWNS,
       getCooldown: haste => 90,
       recommendedCastEfficiency: 0.90,
       isActive: combatant => combatant.hasTalent(SPELLS.OBLITERATION_TALENT.id),
       extraSuggestion: <span>Normally you should be using this off CD.</span>,
+    },   
+    {
+      spell: SPELLS.BREATH_OF_SINDRAGOSA_TALENT,
+      category: CastEfficiency.SPELL_CATEGORIES.COOLDOWNS,
+      getCooldown: haste => 120,
+      recommendedCastEfficiency: 0.90,
+      isActive: combatant => combatant.hasTalent(SPELLS.BREATH_OF_SINDRAGOSA_TALENT.id),
+      extraSuggestion: <span>Normally you should be using this off CD.</span>,
     },
     {
-      spell: SPELLS.PILLAR_OF_FROST, // TO DO: Add support for Gravewarden extension bonus
+      spell: SPELLS.PILLAR_OF_FROST, // TO DO: Add support for Gravewarden extension bonus, Ice Cap reduction
       category: CastEfficiency.SPELL_CATEGORIES.COOLDOWNS,
-      getCooldown: haste => 60,
```

Comment:
You've added Ice Cap reduction under TO DO: even tho it's done? 
Also isn't it purely 1s reduction pr crit?

Reply:
I was afraid I was gonna make the commit to big. So I started on the Ice Cap Reduction, but its not calculating properly. I need to figure out how to count the number of crits between casts.